apps/website/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "@griffel/website",
```

Comment:
Sometimes docusaurus checks for updates and if `package.json` it fails any CLI command.

Reply:
not sure I understood your comment

Reply:
Yeah, missed a few words there, sorry. 

- Docusaurus has update notifier
- It reads `package.json` inside project

https://github.com/facebook/docusaurus/blob/770418f8d262c56cd3de5ba1087f972ab20da617/packages/docusaurus/bin/beforeCli.js#L84-L88

- If it's absent, it throws on any CLI command (`docusaurus build`, `docusaurus start`, etc.)

Reply:
small nit: you could leverage the `description` property to doc that requirement

Reply:
Hmm, good idea, pushed